src/server.jsx

```diff
@@ -46,6 +47,22 @@ if (settings) {
 const supported = new locale.Locales(keys(languages), 'en');
 
 const server = express();
+// Internal proxy to bypass CORS while developing.
+// For the sake of simplicty, we only allow http://localhost:3000/api as the proxied api path
+if (__DEVELOPMENT__ && settings.apiPath.endsWith('http://localhost:3000/api')) {
+  const parsedURL = parseUrl(settings.proxyToApiPath);
+  const serverURL = `${parsedURL.protocol}//${parsedURL.host}`;
+  const instancePath = `${parsedURL.pathname}`;
```

Comment:
```suggestion
  const instancePath = parsedURL.pathname;
```
Small suggestion to make this simpler to read.

Reply:
Fixed.